tools/rpkg/R/relational.R

```diff
@@ -159,19 +159,61 @@ rel_order <- rapi_rel_order
 #' @noRd
 #' @examples
 #' con <- DBI::dbConnect(duckdb())
-#' DBI::dbExecute(con, "CREATE MACRO eq(a, b) AS a = b")
+#' DBI::dbExecute(con, "CREATE OR REPLACE MACRO eq(a, b) AS a = b")
 #' left <- rel_from_df(con, mtcars)
 #' right <- rel_from_df(con, mtcars)
-#' cond <- list(expr_function("eq", list(expr_reference("cyl", left), expr_reference("cyl", right))))
+#' cond <- list(expr_function("=", list(expr_reference("cyl", left), expr_reference("cyl", right))))
```

Comment:
Do we need both changes?

Reply:
ah no, that should be eq. The `create or replace` is just incase. It's also used in a number of the tests